apps/omg/lib/omg/state.ex

```diff
@@ -81,19 +81,15 @@ defmodule OMG.State do
     GenServer.call(__MODULE__, :get_status)
   end
 
-  ### Server
-
   @doc """
   Start processing state using the database entries
   """
   def init(:ok) do
     # Get utxos() is essential for the State and Blockgetter. And it takes a while. TODO - measure it!
     # Our approach is simply blocking the supervision boot tree
     # until we've processed history.
-    {:ok, DB.utxos(), {:continue, :setup}}
-  end
-
-  def handle_continue(:setup, {:ok, utxos_query_result}) do
+    {:ok, utxos_query_result} = DB.utxos()
+    # don't use handle continue, BlockGetter using get_status() in setup
```

Comment:
This comment doesn't apply to following lines but also to above `getting utxos`.
Can you merge it into previous comment? You can also verbosely describe the issue root cause in the :PR: description and refer the :PR: here.